kube_apiserver_metrics/datadog_checks/kube_apiserver_metrics/kube_apiserver_metrics.py

```diff
@@ -62,6 +63,8 @@
     # https://kubernetes.io/docs/reference/using-api/deprecation-policy/#rest-resources-aka-api-objects
     'apiserver_requested_deprecated_apis': 'requested_deprecated_apis',
     # For Kubernetes >= 1.23
+    # https://github.com/kubernetes/kubernetes/pull/103799
+    'apiserver_longrunning_requests': 'longrunning_requests',
```

Comment:
usually we try to remap the new metrics name to the previous name, to ease the life of users (they don't need to update their dashboards or monitors) 

If the new metric return exactly the same value (unit) and is expose with the same type (here gauge), we can remap the metric to the previous name. 

if the unit change, we can use a transformer function to comeback to the previous unit. (when it makes sens)

```suggestion
    'apiserver_longrunning_requests': 'longrunning_gauge',
```

Reply:
@clamoriniere what happens in the case for Kubernetes versions (like 1.23) that had both metrics (the deprecated one, and the new one)

Reply:
So the last applied will override the other. which is fine since the value should be equal